libvirt/image/files/etc/systemd/system/agent-protocol-forwarder.service

```diff
@@ -4,6 +4,9 @@ After=network.target
 
 [Service]
 ExecStart=/usr/local/bin/agent-protocol-forwarder -kata-agent-namespace /run/netns/podns -kata-agent-socket /run/kata-containers/agent.sock
+Restart=on-failure
+RestartSec=5s
+
```

Comment:
Is this needed ?
If it's must, we should also change the common files under `podvm`

Reply:
Yes, without this change, I ended up with a rhel image where agent-protocol-forwarder had exited with failure because of missing /peerpod/daemon.json. Can you please clarify which common files are you referring to ? I could add the restart clauses to other services but from testing, it doesn't look like they are required.

Reply:
The common https://github.com/confidential-containers/cloud-api-adaptor/tree/staging/podvm/files/etc
Further, I think this is a good change to make for all the providers. Can you move the "Restart" setting to a separate PR  for all the providers ?

Reply:
Yeah, I just saw the newer commit, will do. Thanks!